net-misc/rygel/rygel-0.42.0.ebuild

```diff
@@ -0,0 +1,83 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+VALA_MIN_API_VERSION="0.53.2"
+inherit gnome.org meson systemd vala xdg
+
+DESCRIPTION="Rygel is an open source UPnP/DLNA MediaServer"
+HOMEPAGE="https://wiki.gnome.org/Projects/Rygel"
+
+LICENSE="LGPL-2.1+ CC-BY-SA-3.0"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="gtk gtk-doc +introspection man +sqlite tracker test transcode"
+RESTRICT="!test? ( test )"
+
+DEPEND="
+	>=net-libs/gupnp-1.5.2:1.6=[vala] # TODO slot?
+	>=dev-libs/libgee-0.8:0.8
+	>=net-libs/gssdp-1.5.0:1.6=[vala] # TODO slot?
+	>=dev-libs/glib-2.62.0:2
+	>=dev-libs/libxml2-2.7:2
+	>=net-libs/gupnp-av-0.14.1:=
+	>=media-libs/gupnp-dlna-0.9.4:2.0=
+	>=net-libs/libsoup-3:3
+	sqlite? (
+		>=dev-db/sqlite-3.5:3
+		dev-libs/libunistring:=
+	)
+	>=media-libs/gstreamer-1.12:1.0
+	>=media-libs/gst-plugins-base-1.12:1.0
+	media-libs/gstreamer-editing-services:1.0
+	>=media-libs/libmediaart-0.7:2.0[vala]
+	media-plugins/gst-plugins-soup:1.0
+	x11-libs/gdk-pixbuf:2
```

Comment:
I don't see gdk-pixbuf as a direct dependency.

Reply:
https://gitlab.gnome.org/GNOME/rygel/-/blob/master/meson.build#L104 ?

Reply:
Right you are!